engagement_database/data_models.py

```diff
@@ -50,6 +50,8 @@ def __init__(self, text, timestamp, participant_uuid, direction, channel_operato
         :type coda_id: str | None
         :param last_updated: Timestamp this message was last updated in Firestore, or None if it does not yet exist.
         :type last_updated: datetime.datetime | None
+        :param previous_datasets: Datasets which this message originally belonged to/moved from. If None, a return an empty list
```

Comment:
```suggestion
        :param previous_datasets: Datasets which this message originally belonged to/moved from. If None, initialises with an empty list
```

Reply:
I think this is the right thing to do, but I don't see a if previous_datasets is None: previous_datasets = [] in the constructor

Reply:
Was abit conflicted here whether we should return None or an empty list. I think unlike labels previous_datasets should be None unless the message moves. So have created a list here https://github.com/AfricasVoices/Test-Pipeline-Engagement-DB/pull/17/files#diff-834f785e8021c007d4aed1e4aa8bdd5c9deafba3f3687e41bcc6ce0f9b124327R101 only when the message moves. If you agree will update the doc string accordingly.